src/backend/api/handlers/helpers/track_call.py

```diff
@@ -0,0 +1,16 @@
+from flask import request
+
+from backend.common.google_analytics import GoogleAnalytics
+from backend.common.models.api_auth_access import ApiAuthAccess
+from backend.common.run_after_response import run_after_response
+
+
+def track_call_after_response(api_action: str, api_label: str) -> None:
+    auth_key = request.headers.get("X-TBA-Auth-Key", request.args.get("X-TBA-Auth-Key"))
+    auth_owner_id = ApiAuthAccess.get_by_id(auth_key).owner.id()
```

Comment:
It seems silly that we have to go through this logic again since we're already doing this in the `@api_authenticated` decorator. Is there a way to avoid this? The old class implementation let us store stuff in a request-instance variable.